### PR TITLE
Add Tutorial: invariant mass fit of B0

### PR DESCRIPTION
**Pull request by AlexBigO**
Hey @fgrosa @stefanopolitano ! Here is a first version for the tutorial :) Please tell me what can be improved/simplified.

I am not familiar with linking Colab and Github and I guess the notebook I pushed is static. So, to allow you to give direct feedback on the Colab file, here is the original [link](https://colab.research.google.com/drive/1J-QHOLsZjmDj18lb5gybqw8gB-fODLeR?usp=sharing).

Comment: the link towards the GoogleColab file in README.md file will need to be modified once the PR is merged to `dev` and once again once it is merged to `main`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,11 @@ FLexible And REliable Fitting LibrarY is a library to perform fits to distributi
...
 
 [https://flarefly.github.io/flarefly/index.html](https://flarefly.github.io/flarefly/index.html)
+
+## Tutorial
+
+If you want to get familiar with flarefly, the following tutorial is available:
+
+| Type | Link |
+| -------------- | ------------- |
+| Invariant mass fit |  [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/AlexBigO/flarefly/blob/myBranch/tutorials/flarefly_tutorial_Bzero.ipynb) |
```
Comment (fgrosa) [inline README.md:23]: I would specify that it is a tutorial for a binned fit (we will include also the one for the unbinned fit sooner or later ;)) and I would already put it in `main`, with the idea of having soon a new release
```suggestion
| Binned invariant mass fit |  [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/flarefly/flarefly/blob/main/tutorials/flarefly_tutorial_Bzero.ipynb) |
```

**Review by fgrosa — COMMENTED** [review]
Many thanks @AlexBigO! The tutorial looks great :) I just have a couple of very minor comments (see below)

File: tutorials/flarefly_tutorial_Bzero.ipynb [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+        "expo_id = 0     # because background_pdfs[expo_id] = \"expo\""
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rrXKVQgHVBPF"
+      },
+      "source": [
+        "### **2. Define a F2MassFitter instance**\n",
```
Comment (stefanopolitano) [inline tutorials/flarefly_tutorial_Bzero.ipynb:171]: Maybe it is worth mentioning that the name of the F2MassFitter has to be unique for each fitter one wants to use

**Review by stefanopolitano — COMMENTED** [review]
Hi @AlexBigO! Thanks a lot for the work: it looks really nice. I added a couple of minor suggestions.

File: tutorials/flarefly_tutorial_Bzero.ipynb [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "sD-5mNSXmdqn"
+      },
+      "outputs": [],
+      "source": [
+        "%%capture\n",
+        "!curl -L https://cernbox.cern.ch/s/TUhBQIOBMSel5mK/download --output file.root"
```
[the remark this answers is not in the document]
Comment (AlexBigO) [reply tutorials/flarefly_tutorial_Bzero.ipynb:64]: That's what I tried to do initially but then I did not find the right command line to download the file from the git repo. Do you know it? :)

[the remark this answers is not in the document]
Comment (fgrosa) [reply tutorials/flarefly_tutorial_Bzero.ipynb:64]: @AlexBigO this should work:
```
!curl -L https://github.com/flarefly/flarefly/raw/main/tutorials/file_name.root --output file.root
```

[the remark this answers is not in the document]
Comment (AlexBigO) [reply tutorials/flarefly_tutorial_Bzero.ipynb]: I put this line, we'll be able to test it after the merge :)

**Review by fgrosa — APPROVED** [review]

**Pull request merged**